cli/package.json

```diff
@@ -18,11 +18,15 @@
     "/npm-shrinkwrap.json",
     "/oclif.manifest.json"
   ],
+  "engines": {
+    "node": ">=8"
+  },
```

Comment:
Not sure why we need this, also the version looks odd

Reply:
What version should it be instead?

Reply:
18 probably